### ISSUE TITLE
slightly out of date

### ISSUE THREAD

**Issue by sheffler** [mslosarz/nextrtc-example-videochat#21]
Really nice demonstration!

 It almost worked right out of the box, but it seems that this demo needs to provide a configuration of something that changed recently in nextrtc-signaling-server 1.0.0-SNAPSHOT.

As a workaround, I changed the dependency to 0.0.7, like this:

```
		<dependency>
			<groupId>org.nextrtc.signalingserver</groupId>
			<artifactId>nextrtc-signaling-server</artifactId>
			<version>0.0.7</version>
		</dependency>
```

The signaling server is well constructed.  Keep it up.

**Comment by mslosarz** [comment]
I didn't release 1.0.0 yet. I'll do it shortly - I still have some work to
do.

15.08.2017 21:36 "Tom Sheffler" <notifications@github.com> napisał(a):

**Comment by mslosarz** [comment]
I've released version 1.0.0-RC1
